flux-kube/cmd/Makefile.am

```diff
@@ -0,0 +1,4 @@
+all: flux-kube.py
+
+install: flux-kube.py
+	cp flux-kube.py "${FLUX_PREFIX}"/libexec/flux/cmd/
```

Comment:
The current makefile doesn't support `make uninstall` and I'm not sure if it would work with a VPATH build (it might).  If you use the automake syntax, it'll handle all of that for you. Along with setting the proper permissions and inheriting the proper flux command path from flux-core's autotools (in case that ever changes).  See `flux-accounting` for an example: https://github.com/flux-framework/flux-accounting/blob/master/src/cmd/Makefile.am

Reply:
Oh, that's awesome!  Now I fully understand what you meant when you told me about autotools setting this path.  It's even more "auto" than I realized.